packages/web/modals/profile.tsx

```diff
@@ -103,6 +137,133 @@ export const ProfileModal: FunctionComponent<
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
```

Comment:
Try not to use a `useEffect` to set state, this could just be written as a variable 

```
const stargazeAddress =
    address.length > 0
      ? Bech32Address.fromBech32(address, "osmo").toBech32("stars")
      : "";